lib/woothee/crawler.rb

```diff
@@ -88,7 +88,7 @@ def self.challenge_crawlers(ua, result)
       end
     end
     if ua.index('Yeti')
-      if ua.index('http://help.naver.com/robots')
+      if ['http://help.naver.com/robots', 'http://help.naver.com/support/robots.html', 'http://naver.me/bot'].any? { |url| ua.index(url) }
```

Comment:
@tagomoris 
Ah, that's true...
Fixed it.
https://github.com/woothee/woothee-ruby/pull/15/commits/6e3424722cd08117214ce9d389db6cc0a69c3652